.gitattributes

```diff
@@ -3,3 +3,12 @@
 *.jpg -text
```

Comment:
Does it matter that these are listed as text, even though they really binary files.

Reply:
`-text` is unsetting the text attribute on those files to make sure "line endings" don't get normalized.

Reply:
Maybe it would be better to set that using the `binary` macro, which might be clearer than unsetting text.

Reply:
Ah, I see now.  having some mention of binary would make it clearer if possible.